### ISSUE TITLE
Crash when click on help icon

### ISSUE THREAD

**Issue by gokulraam** [uazo/bromite-buildtools#298]
When click on question icon(opens help and feedback in chrome) at right corner of settings page, app crashes?

**Comment by uazo** [comment]
yes, I'm aware of it.  I have to delete the button from all the ui.

theoretically the crash is correct. pressing that button chromium launches an internal procedure for verifying the device settings which it sends to google if the setting relating to sending reports is active.
here it is disabled, but the ui doesn't know about it :)
